apps/entry/serializers.py

```diff
@@ -92,13 +99,64 @@ class Meta:
         fields = ('id', 'exportable', 'data',)
 
 
+class ProjectEntryLabelSerializer(DynamicFieldsMixin, UserResourceSerializer):
+    entry_count = serializers.IntegerField(read_only=True)
```

Comment:
Maybe use another serializer? Because entry_count seems to depend on queryset attribute in viewset.

Reply:
Done